python/dgl/graphbolt/minibatch.py

```diff
@@ -584,15 +584,17 @@ def apply_to(x, device):
                 "node_features",
                 "edge_features",
             ]
-        elif self.seeds is not None and self.compacted_seeds is not None:
+        elif self.seeds is not None:
             # Node/link/edge related tasks.
             transfer_attrs = [
                 "labels",
-                "compacted_seeds",
                 "sampled_subgraphs",
                 "node_features",
                 "edge_features",
             ]
+            # Link/edge related tasks.
```

Comment:
Does this really matter, if compacted_seeds is None, the transfer will not happen, which takes no time.

Reply:
The necessary change is L587, should I delete this change(L595-597)?

Reply:
I meant you don't have to move the compacted_seeds out, but either is fine.